modelzoo/DeepFM/train.py

```diff
@@ -14,6 +14,7 @@
 tf.logging.set_verbosity(tf.logging.INFO)
 print("Using TensorFlow version %s" % (tf.__version__))
 
+# Definition of some constants
 CONTINUOUS_COLUMNS = ["I" + str(i) for i in range(1, 14)]  # 1-13 inclusive
 CATEGORICAL_COLUMNS = ["C" + str(i) for i in range(1, 27)]  # 1-26 inclusive
 LABEL_COLUMN = ["clicked"]
```

Comment:
Use single quotes instead of double quotes

Reply:
revised